src/tests/multihost/ad/test_adparameters.py

```diff
@@ -756,3 +756,56 @@ def test_0018_bz1734040(multihost, adjoin, create_aduser_group):
         client.restore_sssd_conf()
 
         assert find.search(log), "Expected log record is missing."
+
+    @pytest.mark.tier1_3
+    def test_0019_bz2070189(self, multihost, adjoin):
+        """
+        :title: sssd error triggers backtrace
+        :description:
+            Global Catalog lookups should not update the kdcinfo files
+        :id: 91bd8845-20e4-4fde-9914-73b08d44c195
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2070189
+        :steps:
+            1. setup AD environment
+            2. Join linux machine into AD domain
+            3. Set the debug_level in sssd.conf for AD domain to 1,
+               make sure backtrace dump is enabled,
+               debug_backtrace_enabled=True
+               make sure that krb5_use_kdcinfo=True
+            4. clear the cache sss_cache -E
+            5. lookup AD user `id administrator@domain`
+        :expectedresults:
+            1. null
+            2. null
+            3. null
+            4. null
+            5. Backtrace is NOT triggered
+        """
+        adjoin(membersw='adcli')
+        backup = 'cp -f /etc/sssd/sssd.conf /etc/sssd/sssd.conf.orig'
+        restore = 'cp -f /etc/sssd/sssd.conf.orig /etc/sssd/sssd.conf'
+        multihost.client[0].run_command(backup)
+        client = sssdTools(multihost.client[0])
+        domain_name = multihost.ad[0].domainname
+        domain_log = '/var/log/sssd/sssd_%s.log' % domain_name
+        domain_sec_name = client.get_domain_section_name()
+        dom_section = 'domain/%s' % domain_sec_name
+        sssd_params = {'domains': 'files, %s' % domain_sec_name}
+        client.sssd_conf('sssd', sssd_params)
+        domain_params = {'debug_level': '1', 'krb5_use_kdcinfo': 'True',
+                         'debug_backtrace_enabled': 'True'}
+        client.sssd_conf(dom_section, domain_params)
+        file_section = 'domain/files'
+        file_params = {'id_provider': 'files'}
+        client.sssd_conf(file_section, file_params)
+        multihost.client[0].run_command('id')
+        multihost.client[0].run_command('sss_cache -E')
```

Comment:
Possibly client.clear_sssd_cache() can be used here to restart sssd clearing caches and logs.